specification/ai/OpenAI.Assistants/run_steps/models.tsp

```diff
@@ -165,3 +165,181 @@ model RunStepError {
   @doc("The human-readable text associated with this error.")
   message: string;
 }
+
+//
+// The following types facilitate streaming use.
+//
+
+/** Represents a run step delta i.e. any changed fields on a run step during streaming. */
+@added(ServiceApiVersions.vFuturePlaceholder)
+model RunStepDeltaChunk {
+  /** The identifier of the run step, which can be referenced in API endpoints. */
+  id: string;
+
+  /** The object type, which is always `thread.run.step.delta`. */
+  object: "thread.run.step.delta";
+
+  /** The delta containing the fields that have changed on the run step. */
+  delta: RunStepDeltaChunkObject;
+}
+
+/** Represents the delta payload in a streaming run step delta chunk. */
+@added(ServiceApiVersions.vFuturePlaceholder)
+model RunStepDeltaChunkObject {
+  /** The details of the run step. */
+  @encodedName("application/json", "step_details")
+  stepDetails?: RunStepDeltaDetail[];
+}
+
+/** Represents a single run step detail item in a streaming run step's delta payload. */
+@discriminator("type")
+@added(ServiceApiVersions.vFuturePlaceholder)
+model RunStepDeltaDetail {
+  /** The object type for the run step detail object. */
+  type: string;
+}
+
+/** Represents a message creation within a streaming run step delta. */
+@added(ServiceApiVersions.vFuturePlaceholder)
+model RunStepDeltaMessageCreation extends RunStepDeltaDetail {
+  /** The object type, which is always "message_creation." */
+  type: "message_creation";
+
+  /** The message creation data. */
+  @encodedName("application/json", "message_creation")
+  messageCreation: RunStepDeltaMessageCreationObject;
```

Comment:
This field appears to be "wire-optional" according to this [line](https://github.com/openai/openai-openapi/blob/fbe52bd1bf7d16ea972c85f805a83bbb3180d539/openapi.yaml#L9592-L9594)